indexer/indexer.go

```diff
@@ -838,28 +838,48 @@ func (ix *Indexer) FetchAndIndexRepo(ctx context.Context, job *crawlWork) error
 	}
 
 	var rebase *comatproto.SyncSubscribeRepos_Commit
-	for _, job := range job.catchup {
-		if job.evt.Rebase {
-			rebase = job.evt
+	var rebaseIx int
+	for i, j := range job.catchup {
+		if j.evt.Rebase {
+			fmt.Println("REBASE AT: ", i, len(job.catchup))
```

Comment:
Do we have a logger we can use for this?

Reply:
oh whoops, yes.